pkg/metadata/host/host_windows.go

```diff
@@ -172,3 +175,46 @@ func Pids() ([]int32, error) {
 	}
 
 }
+
+// GetNetworkInfo returns host specific network configuration.
+// At this time, only information queried is the ephemeral port range
+func GetNetworkInfo() (*NetworkInfo, error) {
+	//
+	// note that this will return the ephemeral port range for ipv4/tcp
+	// Windows allows individual configuration for ipv4/6 and udp/tcp.
+	cmd := exec.Command("netsh", "int", "ipv4", "show", "dynamicport", "tcp")
```

Comment:
Is this information available via the registry? I don't love having to shell out and do regexp matching.

Reply:
Not that I was able to find, or at least not by default.
If you change the settings, then it's stored in the registry.  But if it's just defaults, if not.
So we could look in the registry, and if it's not there do the shell out.  I hesitate to just hard-code the defaults as they're liable to change from osver to osver.

Reply:
I'm guessing that most folks don't override, so we would be shelling out in most cases anyways. This method is OK then.

Reply:
I'm not sure I like the shell-out approach but I guess it's marginally better than hardcoding all windows variant ranges. 

PS: There may be a discrepancy in ipv6 and ipv4 ports that we may need to take into account.